beacon-chain/sync/subscriber_light_client.go

```diff
@@ -0,0 +1,44 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/core/feed"
+	statefeed "github.com/prysmaticlabs/prysm/v5/beacon-chain/core/feed/state"
+	light_client "github.com/prysmaticlabs/prysm/v5/consensus-types/light-client"
+	"google.golang.org/protobuf/proto"
+)
+
+func (s *Service) lightClientFinalityUpdateSubscriber(_ context.Context, msg proto.Message) error {
+	update, err := light_client.NewWrappedFinalityUpdate(msg)
+	if err != nil {
+		return err
+	}
+
+	log.Info("LC: storing new finality update in p2p subscriber")
+	s.lcStore.LastLCFinalityUpdate = update
```

Comment:
Shouldn't we make sure this update is better than the one in our local store ?

Reply:
Good point, we definitely should